### PR TITLE
Added LOGICAL_SIZE measurement

### PR DESCRIPTION
**Pull request by wywywywy**
Missing LOGICAL_SIZE measurement as mentioned in issue #32

### CODE REVIEW

File: src/query_parameter_ctrl.ts [hunk 1]
```diff
@@ -313,7 +314,7 @@ export class GrafanaMongoDbQueryParameterCtrl {
       );
       $scope.target.dimensionId =
         HUMANITZED_DIMENSIONS[$scope.target.metricId][
-          $scope.target.dimensionName
+        $scope.target.dimensionName
```
Comment (CapChrisCap) [inline src/query_parameter_ctrl.ts:317]: Can you please remove the formatting?

File: src/query_parameter_ctrl.ts [hunk 2]
```diff
@@ -346,7 +347,7 @@ export class GrafanaMongoDbQueryParameterCtrl {
     };
 
     if (!$scope.onChange) {
-      $scope.onChange = () => {};
+      $scope.onChange = () => { };
```
Comment (CapChrisCap) [inline src/query_parameter_ctrl.ts:350]: Can you please remove the formatting?

**Review by CapChrisCap — CHANGES_REQUESTED** [review]
Awesome, thanks for the MR. Can you please fix the minor changes? Then we can merge your change! 👍

**Comment by wywywywy** [comment]
Sorry not sure what happened there.  I didn't think I had auto-formatting switched on.

Anyway, new commit done.

**Comment by wywywywy** [comment]
The updated instruction has a problem.

The **;mongodb-atlas-datasource** part is missing.

`-e GF_INSTALL_PLUGINS="https://github.com/valiton/grafana-mongodb-atlas-datasource/releases/download/v2.0.0/mongodb-atlas-datasource.zip;mongodb-atlas-datasource"`

**Comment by CapChrisCap** [comment]
Thanks for mentioning this! Directly fixed the README.

**Pull request merged**